frontend/test/__support__/e2e/commands.js

```diff
@@ -43,10 +43,16 @@ Cypress.Commands.add("button", button_name => {
   cy.findByRole("button", { name: button_name });
 });
 
-Cypress.Commands.add("createDashboard", name => {
-  cy.log(`Create a dashboard: ${name}`);
-  cy.request("POST", "/api/dashboard", { name });
-});
+Cypress.Commands.add(
+  "createDashboard",
+  (name, { collection_position = null } = {}) => {
+    cy.log(`Create a dashboard: ${name}`);
+    cy.request("POST", "/api/dashboard", {
+      name,
+      collection_position,
+    });
+  },
+);
```

Comment:
Extended `cy.createDashboard` command a bit, so we can specify extra parameters for creation requests. In my case, I wanted to create pinned items with an API request.

Should we maybe plan a refactoring for this command, so we can unify `createDashboard` and `createQuestion` APIs? Question command accepts an object with question details as a single argument. Dashboard command was accepting only the `name` string

Reply:
`createDashboard` accepted only `name` because it needed exactly that for 99% of the tests. In fact, even name could've been auto-generated because it didn't matter. Now that the need for more options arose, we can definitely expand it.

However, this will create a merge conflict because custom commands have already been refactored on the release branch. Let me think what's the best way to deal with this.